packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -95,9 +101,10 @@ def generate_tokens(self):
         apiview = self._generate_tokens(pkg_root_path, pkg_name, version)
         if apiview.Diagnostics:
             # Show error report in console
-            print("************************** Error Report **************************")
-            for m in self.module_dict.keys():
-                self.module_dict[m].print_errors()
+            if not self.hide_report:
+                print("************************** Error Report **************************")
+                for m in self.module_dict.keys():
+                    self.module_dict[m].print_errors()
```

Comment:
Does using `hide_report` also change the script exit code?
For that matter - does having errors in the report change the exit code?

Reply:
@annatisch : Currently lint errors doesn't change exit code since we consider them as soft errors only. When I change this to hard error, I will handle error code when running from API tool.